reference/docs-conceptual/AIShell/how-to/mcp-support.md

```diff
@@ -0,0 +1,159 @@
+  executing it. The user can review and choose to run it manually by pressing Enter.
+- `run_command_in_terminal` - This tool allows you to execute shell commands in a persistent
+  PowerShell session, preserving environment variables, working directory, and other context across
+  multiple commands.
+- `get_command_output` - Get the output of a command previously started with
+  `run_command_in_terminal`.
+
+> [!NOTE]
+> The built-in tools rely on the side-car experience with a connected PowerShell session and provide
+> enhanced context awareness and automation capabilities.
+
+Here is a simple demo showing how you can have AI Shell run commands on your behalf using the
+`run_command_in_terminal` tool:
+
+![Have the MCP run a command in terminal for you.][09]
+
+This example shows how additional context is provided to AI Shell to improve results:
+
+![Getting more context with built-in tools.][07]
+
+You can also use the `get_terminal_content` tool to get the content from the connected terminal and
+provide it to AI Shell to help it understand what you are trying to do:
+
+![Getting content from the terminal for output created before AI Shell started.][08]
+
+## Find and configure MCP servers
+
+An MCP server is an application that runs locally or is hosted remotely and accessed through an
+endpoint URL. Many MCP servers are open source and available for free. Local MCP server apps are
+typically built using Node.js and can installed using `npm`. You can find a list of available MCP
+servers at [Awesome MCP Servers][03].
+
+Before you can use an MCP server (Local or remote) you must add it to the `mcp.json` configuration
+file. In some cases, you might not need to install the local MCP before using it. However, the MCP
+server will be installed the first time you start AI Shell. This can take several minutes. For the
+best experience, preinstall the MCP server. Consult the documentation for your specific MCP server
+for installation instructions. Remote MCP servers don't need to be installed. You just need to add
+them to the `mcp.json` file.
+
+### Install a local MCP server
+
+The following example installs the `@modelcontextprotocol/server-everything` and
+`@modelcontextprotocol/server-filesystem` MCP servers. These servers are Node.js. You must have
+`Node.js` installed to use these MCP servers.
+
+To install the MCP servers, you can use the following commands:
+
+```powershell
+npm install -g @modelcontextprotocol/server-everything
+npm install -g @modelcontextprotocol/server-filesystem
+```
+
+For more information about their command and capabilities, see the following documentation:
+
+- [Everything MCP Server][04]
+- [Filesystem MCP Server][05]
+
+### Add MCP Servers
+
+To add an MCP server, create an `mcp.json` file in `$HOME\.aish\` folder. The following example
+shows two local MCP servers, `everything` and `filesystem`, and two remote MCP server. You can add
+as many MCP servers as you want.
```

Comment:
Non-blocking for this PR, but it might be worth drafting a page about defining `mcp.json` for AI Shell specifically. Presumably the JSON adheres to a schema, which hopefully we can link to and has separately maintained documentation.